doc/tor.md

```diff
@@ -5,18 +5,33 @@ It is possible to run Bitcoin Core as a Tor onion service, and connect to such s
 The following directions assume you have a Tor proxy running on port 9050. Many distributions default to having a SOCKS proxy listening on port 9050, but others may not. In particular, the Tor Browser Bundle defaults to listening on port 9150. See [Tor Project FAQ:TBBSocksPort](https://www.torproject.org/docs/faq.html.en#TBBSocksPort) for how to properly
 configure Tor.
 
+## How to see information about your Tor configuration via Bitcoin Core
+
+There are several ways to see your local onion address in Bitcoin Core:
+- in the debug log (grep for "tor:" or "AddLocal")
+- in the output of RPC `getnetworkinfo` in the "localaddresses" section
+- in the output of the CLI `-netinfo` peer connections dashboard
+
+You may set the `-debug=tor` config logging option to have additional
+information in the debug log about your Tor configuration.
+
 
 ## 1. Run Bitcoin Core behind a Tor proxy
 
 The first step is running Bitcoin Core behind a Tor proxy. This will already anonymize all
 outgoing connections, but more is possible.
 
-	-proxy=ip:port  Set the proxy server. If SOCKS5 is selected (default), this proxy
+	-proxy=ip:port  Set a proxy server. If SOCKS5 is selected (default), this proxy
```

Comment:
What does it mean "If SOCKS5 is selected"? How does one select SOCKS5 or what other options are there to select from?

I think this should be removed:

```suggestion
      -proxy=ip:port  Set a proxy server. This proxy
```

as `-proxy=` only supports SOCKS5 proxies.

Reply:
I can drop this, but it was existing before this PR.

Reply:
Dropping this commit.